playback/service/src/main/java/de/danoeh/antennapod/playback/service/PlaybackService.java

```diff
@@ -228,9 +228,12 @@ public void onCreate() {
         stateManager = new PlaybackServiceStateManager(this);
         notificationBuilder = new PlaybackServiceNotificationBuilder(this);
 
-        registerReceiver(autoStateUpdated, new IntentFilter("com.google.android.gms.car.media.STATUS"));
+        ContextCompat.registerReceiver(this, autoStateUpdated,
+                new IntentFilter("com.google.android.gms.car.media.STATUS"), ContextCompat.RECEIVER_EXPORTED);
+        ContextCompat.registerReceiver(this, shutdownReceiver,
+                new IntentFilter(PlaybackServiceInterface.ACTION_SHUTDOWN_PLAYBACK_SERVICE),
+                ContextCompat.RECEIVER_NOT_EXPORTED);
         registerReceiver(headsetDisconnected, new IntentFilter(Intent.ACTION_HEADSET_PLUG));
```

Comment:
@ByteHamster I am wondering whether we need a handle this for unplugging and plugging in headphones for continuous playback

Reply:
Sorry, I don't really understand what you mean with this.